docs/versioned_docs/version-v0.3/concepts/credentials/username-email-password.mdx

```diff
@@ -287,7 +287,7 @@ Assuming your traits schema is as follows:
 
 And an identity registers with the following JSON payload (more on registration
 in
-[Selfservice Registration](../self-service/flows/user-login-user-registration.md)):
+[Selfservice Registration](../self-service/flows/user-login-user-registration)):
```

Comment:
[Selfservice Registration](../../self-service/flows/user-login-user-registration)):

Reply:
@jfcurran what do you mean here?

@vinckr : http://archive.download.redhat.com/pub/redhat/linux/7.3/emea/doc/RH-DOCS/rhl-gsg-en-7.3/s1-navigating-cd.html

```suggestion
[Selfservice Registration](../../self-service/flows/user-login-user-registration)):
```